From 3d953c4bf69566c5fb290bd34539d994f2aa0c17 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:22:00 -0400
Subject: [cpufreq] Fix white space in __cpufreq_remove_dev()

Message-id: <1409235879-1283-27-git-send-email-prarit@redhat.com>
Patchwork-id: 89362
O-Subject: [RHEL7.1 PATCH BZ 1134639 026/185] cpufreq: Fix white space in __cpufreq_remove_dev()
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit edab2fbc21b9eb37007ad8bffe1159d536bbb451
Author: Viresh Kumar <viresh.kumar@linaro.org>
Date:   Tue Aug 20 12:08:22 2013 +0530

    cpufreq: Fix white space in __cpufreq_remove_dev()

    Align closing brace '}' of an if block.

    [rjw: Subject and changelog]
    Signed-off-by: Viresh Kumar <viresh.kumar@linaro.org>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/cpufreq.c b/drivers/cpufreq/cpufreq.c
index ff1d9a7..0e1eded 100644
--- a/drivers/cpufreq/cpufreq.c
+++ b/drivers/cpufreq/cpufreq.c
@@ -1221,7 +1221,7 @@ static int __cpufreq_remove_dev(struct device *dev,
       __func__);
     return ret;
    }
- }
+  }
 
   if (!frozen) {
    lock_policy_rwsem_read(cpu);
-- 
1.7.1